graphrag/index/graph/extractors/claims/claim_extractor.py

```diff
@@ -37,7 +38,7 @@ class ClaimExtractorResult:
 class ClaimExtractor:
     """Claim extractor class definition."""
 
-    _llm: CompletionLLM
+    _llm: OpenAITextChatLLMInstance
```

Comment:
In the future, how do we change this for a non-openai model? is there a more generic Protocol we can use here instead of a OpenAITextChat ?

Reply:
Yep, this will be part of the next PR. We made the types fit a little too nicely to OpenAI in fnllm, and I'm going to add an intermediate level abstraction that's not openai-bound.

Reply:
Yep, this will be part of the next PR. We made the types fit a little too nicely to OpenAI in fnllm, and I'm going to add an intermediate level abstraction that's not openai-bound.